README.md

```diff
@@ -11,8 +11,9 @@ Included in the repository is a PoC for running a plain wasi host (ie. no extra
 
 ### Community
 
-Come join us on our [slack channel #runwasi](https://cloud-native.slack.com/archives/C04LTPB6Z0V)
+- Come join us on our [slack channel #runwasi](https://cloud-native.slack.com/archives/C04LTPB6Z0V)
```

Comment:
It might be useful to point where to join the Slack, I've remember being a new and not knowing how to get the invite:

https://slack.cncf.io/

Reply:
Doesn't the link already work?

Reply:
If you are non already signed up it doesn't work iircc